src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/IridaOAuthErrorHandler.java

```diff
@@ -30,11 +30,14 @@ public void handleError(ClientHttpResponse response) throws IOException {
 		logger.trace("Checking error type " + statusCode.toString());
 		switch (statusCode) {
 		case UNAUTHORIZED:
-			logger.trace("Throwing new IridaOAuthException for this error");
-			throw new IridaOAuthException("User is unauthorized for this service", remoteAPI);
+			logger.trace("Throwing new IridaOAuthException (Unauthorized) for this error");
+			throw new IridaOAuthException("User is unauthorized for this service", statusCode, remoteAPI);
 		case FORBIDDEN:
+			logger.trace("Throwing new IridaOAuthException (Forbidden) for this error");
+			throw new IridaOAuthException("User is forbidden from accessing this service", statusCode, remoteAPI);
```

Comment:
Can you change `service` here to `resource`.

Reply:
Updated in 4d292bf1bc247ddd4230213ae0fed6957cf444a8